static/css/style.css

```diff
@@ -1684,22 +1683,44 @@ input.input-has-error {
 }
 
 .banner-logo {
-  background-position: center center;
+  background-position: top center;
   background-size: contain;
-  height: 72px;
-  width: 72px;
+  height: 60px;
+  width: 60px;
+  min-height: 60px;
+  min-width: 60px;
   background-repeat: no-repeat;
-  margin: 0 .25em auto 1em;
+  margin: 0 1.25rem 0 2.5rem;
 }
 
 .banner-relay-logo {
+  background: none;
+  position: relative;
+}
+
+.banner-relay-logo::after {
```

Comment:
This is a work-around to get the icon to appear 60x60, even though elements of it "break out" of that ratio. 

![image](https://user-images.githubusercontent.com/2692333/120381001-859fac00-c2e7-11eb-8ca1-50e6609ce19e.png)